src/components/scene/inspector.js

```diff
@@ -12,12 +12,35 @@ module.exports.Component = registerComponent('inspector', {
   },
 
   init: function () {
+    this.loadingMessage = 'Loading Inspector';
     this.onKeydown = bind(this.onKeydown, this);
     this.onMessage = bind(this.onMessage, this);
+    this.initOverlay();
     window.addEventListener('keydown', this.onKeydown);
     window.addEventListener('message', this.onMessage);
   },
 
+  initOverlay: function () {
+    this.overlayEl = document.createElement('div');
+    this.overlayEl.style.backgroundColor = '#222';
+    this.overlayEl.style.opacity = '0.90';
+    this.overlayEl.style.position = 'absolute';
+    this.overlayEl.style.width = '800px';
+    this.overlayEl.style.height = '600px';
+    this.overlayEl.style.textAlign = 'left';
+
+    this.loadingMessageEl = document.createElement('div');
```

Comment:
I'd give it `text-transform: capitalize`, increased font size, and `monospace` font-family.